### ISSUE TITLE
Placeholder link style contrast refactor

### ISSUE THREAD

**Issue by Izzy-Snyder** [processing/p5.js-web-editor#4319]
### Increasing Access

This feature would ensure that links without specified styles will by default have high enough contrast with the main background colors of each theme.

### Feature enhancement details

Links in the console area of the editor use the placeholder link color, which is `inactive-text-color` at rest and `heavy-text-color ` on hover. 

This makes the links far too low contrast in the light theme (1.9:1 ratio):

<img width="1556" height="340" alt="Image of the console in the light theme with the low contrast link color" src="https://github.com/user-attachments/assets/53db582f-4315-42bc-8062-9797a1fee838" />

Using `heavy-text-color` at rest would give a 15:1 contrast ratio:
<img width="1494" height="338" alt="Image of the console in the light theme with the high contrast link color" src="https://github.com/user-attachments/assets/b9c31c9d-e6b5-4798-912f-b29246254971" />

My proposed fix is to change the placeholder link styling to `heavy-text-color` by default. The link should also have an underline by default that becomes thicker on hover (like the change in #3560 ). The thickness should be 0.1em. 

Changing the links in dark and high contrast mode to `heavy-text-color` would also increase the contrast (though the contrast is ok as is), so this change would not decrease accessibility.

Changes:

- [ ] in _placeholders.scss, update the link styling
- [ ] change color from `inactive-text-color' to `heavy-text-color`
- [ ] change `text-decoration: none` to `text-decoration: underline` for the default link state
- [ ] add `text-decoration-thickness: 0.1em;` to the hover state.

**Comment by SableRaf** [comment]
Seems like a good problem to fix! Maybe using a color that contrasts more with the running text of the error so it's easier to parse visually? With `heavy-text-color` it's harder to distinguish from the rest of the text, the way it is in the current version.

**Comment by GnanaloshiniA27** [comment]
Hi! I'd like to work on this issue. Could you please assign it to me? Thank you!

**Comment by Izzy-Snyder** [comment]
Just assigned you @GnanaloshiniA27! Thanks for your interest in working on the issue.

And thank you for the feedback @SableRaf on making sure the links stand out - @GnanaloshiniA27 how much experience with CSS/sass do you have? If you feel comfortable doing so, you could pick a different contrasting link color for each theme to use for placeholder links (feel free to reference https://github.com/processing/p5.js-web-editor/pull/4159 for existing high contrast link colors!)

If that feels too complex as a starting point, feel free to start by just changing to heavy text color and adding the underline, and post some screenshots!
